pages.nl/common/chmod.md

```diff
@@ -5,28 +5,32 @@
 
 - Geef een gebruiker ([u]ser) die het bestand beheert het recht om deze uit te voeren (e[x]ecute):
 
-`chmod u+x {{bestand}}`
+`chmod u+x {{pad/naar/bestand}}`
 
 - Geef de gebruiker ([u]ser) het recht om een bestand of map te lezen ([r]ead) en schrijven ([w]rite):
 
-`chmod u+rw {{bestand_of_map}}`
+`chmod u+rw {{pad/naar/bestand_of_map}}`
```

Comment:
ref: [common-arguments.md](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)
```suggestion
`chmod u+rw {{pad/naar/bestand_of_directory}}`
```

Reply:
Directory in Dutch is map.